packages/KCard/KCard.vue

```diff
@@ -16,6 +17,12 @@
         <slot name="actions"/>
       </div>
     </div>
+    <div class="k-card-help-text">
```

Comment:
```suggestion
    <div v-if="helpText || $scopedSlots.helpText" class="k-card-help-text">
```

Better to not display the div if we don't need to.